libs/hdf-converters/src/conveyor-mapper.ts

```diff
@@ -0,0 +1,244 @@
+import {ExecJSON} from 'inspecjs';
+import _ from 'lodash';
+import {version as HeimdallToolsVersion} from '../package.json';
+import {BaseConverter, ILookupPath, MappedTransform} from './base-converter';
+import {
+  DEFAULT_STATIC_CODE_ANALYSIS_NIST_TAGS,
+} from './utils/global';
+enum scannerType {
+  Moldy = 'Moldy',
+  Stigma = 'Stima', 
+  CodeQuality = 'CodeQuality',
+  Default = 'Default'
+}
+function createDescription(
+  data: Record<string, unknown>,
+  score: number,
+  date: string,
+  type: string,
+  endTime:string
+): Record<string, unknown> {
+  let desc = '';
+  if (type === scannerType.Moldy || type == scannerType.Stigma) {
+    desc =
+     'title_text:' +
+      (_.get(data, 'title_text') as string) +
+      '\nbody:' +
+      _.get(data, 'body') +
+      '\nbody_format:' +
+      (_.get(data, 'body_format') as string) +
+      '\nclassificaton:' +
+      (_.get(data, 'classification') as string) +
+      '\ndepth:' +
+      (_.get(data, 'depth') as string) +
+      '\nheuristic_heur_id:' +
+      (_.get(data, 'heuristic.heur_id') as string) +
+      '\nheuristic_score:' +
+      (_.get(data, 'heuristic.score') as string) +
+      '\nheuristic_name:' +
+      (_.get(data, 'heuristic.name') as string) +
+      '\n';
+  } else if (type === scannerType.CodeQuality) {
+    desc =
+      'body:' +
+      _.get(data, 'body') +
+      '\nbody_format:' +
+      (_.get(data, 'body_format') as string) +
+      '\nclassificaton:' +
+      (_.get(data, 'classification') as string) +
+      '\ndepth:' +
+      (_.get(data, 'depth') as string) +
+      '\ntitle_text:' +
+      (_.get(data, 'title_text') as string) +
+      '\n';
+  } else {
+    desc = JSON.stringify(data);
+  }
+  return {
+    status: determineStatus(score),
+    code_desc: desc,
+    start_time: date,
+    run_time:  new Date(endTime).valueOf() - new Date(date).valueOf()
+  };
+}
+
+function childrenfinder(currLevel: Record<string, unknown>): string[][] {
+  const arr: string[][] = [];
+  for (const [sha, file] of Object.entries(currLevel)) {
+    if (_.has(file, 'name')) {
+      const name: string = _.get(file, 'name[0]') || '';
+      arr.push([sha, name]);
+    }
+    if (_.has(file, 'children')) {
+      const nextLevel: Record<string, unknown> = _.get(file, 'children') || {};
+      for ( const element of childrenfinder(nextLevel)) {
+        arr.push(element);
+      };
+    }
+  };
+  return arr;
+}
+function sha2filenameMapper(
+  results: Record<string, unknown>
+): Record<string, unknown> {
+  const toplevel = _.get(results, 'api_response.file_tree') as Record<
+    string,
+    unknown
+  >;
+  const shaMappings = {};
+  const arr = childrenfinder(toplevel)
+  arr.forEach((value) => {
```

Comment:
use decomposition to extract variable names instead of doing value[0] etc.

please fix everywhere possible